swift-package/build_package_macOS.sh

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
I think its better to move it to build.py

Reply:
ok

Reply:
ok